docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -613,6 +613,21 @@ oc logs -l app=openstackansibleee -f --max-log-requests 20
 oc wait --for condition=Ready openstackdataplanedeployment/openstack-pre-adoption --timeout=10m
 ----
 
+. If the validation deployment failed then check the ansible execution logs to
+see which check failed and follow the instructions below:
+
+* if the hostname validation failed then check that the hostname of the EDPM
+node is correctly listed in the `OpenStackDataPlaneNodeSet`
+
+* if the kernel argument check failed then make sure that the
+`OpenStackDataPlaneNodeSet` has the same kernel argument configuration in
+`edpm_kernel_args` and `edpm_kernel_hugepages` variables than what is used in
+the 17 node.
+
+* if the tuned profile check failed then make sure that the
+`edpm_tuned_profile` variable in the `OpenStackDataPlaneNodeSet` configured to
+use the same profile as it used on the 17 node.
```

Comment:
same profile as set on the (source) OSP 17 node.

Reply:
Applied